webpack/ForemanTasks/Components/TasksTable/TasksTableReducer.js

```diff
@@ -0,0 +1,63 @@
+import Immutable from 'seamless-immutable';
+import { combineReducers } from 'redux';
+import { createTableReducer } from 'foremanReact/components/common/table';
+import {
+  TASKS_CONTROLLER,
+  TASKS_SUCCESS,
+  SET_SORT_TASKS_TABLE,
+  SET_PAGINATION_TASKS_TABLE,
+} from './TasksTableConstants';
+
+const initialState = Immutable({
+  query: null,
```

Comment:
Shouldn't it suppose to be removed once it using the base reducer?

Reply:
Not sure what you mean, but I see now that ` query: null,` has no meaning